examples/braket_features/Noise_models/Noise_models_on_Rigetti.ipynb

```diff
@@ -1,1053 +1,989 @@
+            "metadata": {},
+            "source": [
+                "Braket provides access to hardware providers' reported calibration data. \n",
+                "This can be used to construct noise models to approximate the behavior of the QPU when running circuits on a noisy simulator.\n",
+                "In this tutorial, we focus on local noise models with no crosstalk interactions. Real devices can have crosstalk and unexpected effects that can further degrade the results.\n",
+                "\n",
+                "The Ankaa-2 calibration data is available on the Braket devices page. Under qubit specs, the calibration data include the qubit index, with corresponding values for the $T_1$, $T_2$, fidelity from randomized benchmarking (fRB),  fidelity from simultaneous randomized benchmarking (fsRB), and readout fidelity (fRO).\n",
+                "Under \"edge specs\", the data includes the RB fidelity for two qubit gates for each connected edge in the device topology."
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "38001b96-faf7-49f0-b5a5-2d52820fcfc9",
+            "metadata": {},
+            "source": [
+                "**One-qubit calibration data (Qubit specs)**\n",
+                "<div align=center><img src=\"ankaa2_qubit_specs.png\"/></div>\n",
+                "\n",
+                "\n",
+                "**Two-qubit calibration data (Edge specs)**\n",
+                "<div align=center><img src=\"ankaa2_edge_specs.png\"/></div>"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "276d680f-887d-4929-bcaa-e276a6303496",
+            "metadata": {},
+            "source": [
+                "We can programmatically access all the calibration data with the Braket SDK. First we load the AwsDevice using the ARN for Rigetti Ankaa-2."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 2,
+            "id": "a55ea9d5-05f3-4da5-a305-a391e914d012",
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "rigetti = AwsDevice(Devices.Rigetti.Ankaa2)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "a8a42ae1-51ac-4bf1-bd6a-81ffd982b5f8",
+            "metadata": {},
+            "source": [
+                "The properties dictionary contains keys \"provider\" and \"specs\", with \"1Q\" and \"2Q\" keys for the one- and two-qubit calibration data. "
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 3,
+            "id": "153b5b63-8205-4a7e-bb18-0305ef428f36",
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "one_qubit_data = rigetti.properties.standardized.oneQubitProperties\n",
+                "two_qubit_data = rigetti.properties.standardized.twoQubitProperties"
```

Comment:
The markdown cell above still references `["provider"]["specs"]` from the old code - needs to be updated.